setup.py

```diff
@@ -30,14 +30,15 @@
         'pyyaml',
         'requests',
         'funcsigs>=1.0.2',
+        'packaging'
```

Comment:
Why `packaging` is needed? Where is it used?

Reply:
It is used for detecting the `TF` version, since the toggling warning code itself causes deprecation warning in `TF==1.14.0`, to support the compatibility I added it to handle `TF==1.14.0` case.